src/chrome/content/rules/thecustomizewindows.com.xml

```diff
@@ -1,38 +1,20 @@
-<ruleset name="The Customize Windows.com">
-
-	<!--	Direct rewrites:
-				-->
-<target host="thecustomizewindows.com" />
-<target host="www.thecustomizewindows.com" />
-
-	<!--	Complications:
-					-->
-<target host="thecustomizewindows.net" />
-<target host="www.thecustomizewindows.net" />
-
-		<!-- needed for debugging and dev works related to HSTS -->
-
-		<!--exclusion pattern="^http://cloud\.thecustomizewindows\.com/"/-->
-
-			<!--test url="http://cloud.thecustomizewindows.com/" /-->
-
-<!-- Akamai CDN -->
-
-<securecookie host="^a248\.e\.akamai\.net$" name=".+"/>
-
-<!-- allow trusted services -->
-
-<securecookie host="^cdn\.goroost\.com$" name=".+"/>
-<securecookie host="^www\.googletagservices\.com$" name=".+"/>
-<securecookie host="^tpc\.googlesyndication\.com$" name=".+"/>
-<securecookie host="^googleads\.g\.doubleclick\.net$" name=".+"/>
-
-
-<!-- forcing higher security for currently unused dot net domain -->
-
-<rule from="^http://(?:www\.)?thecustomizewindows\.net/" to="https://thecustomizewindows.com/"/>
+<ruleset name="thecustomizewindows.com">
+	
+	<target host="thecustomizewindows.com" />
+	<target host="www.thecustomizewindows.com" />
+	
+	<target host="thecustomizewindows.net" />
+	<target host="www.thecustomizewindows.net" />
+	
+	<securecookie host="^thecustomizewindows-3470\.e\.kxcdn\.com$" name=".+"/>
+	<securecookie host="^cdn\.goroost\.com$" name=".+"/>
+	<securecookie host="^www\.googletagservices\.com$" name=".+"/>
+	<securecookie host="^tpc\.googlesyndication\.com$" name=".+"/>
+	<securecookie host="^googleads\.g\.doubleclick\.net$" name=".+"/>
+	
+	<rule from="^http://(?:www\.)?thecustomizewindows\.net/" to="https://thecustomizewindows.com/"/>
```

Comment:
Use `(www\.)?` instead on `(?:www\.)?`
Please also document why this redirect is needed.